### ISSUE TITLE
[Keyboard a11y] View playlist page

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/avalon#6358]
### Description

From accessibility checks performed in https://github.com/avalonmediasystem/avalon/issues/6312, the following keyboard navigation issues were identified for the view playlist page;
- both the `Markers` and `Source Item Details` accordion panel are not reachable with `Tab` navigation in view playlist page. To observe this behavior, use `Tab` navigation in the view playlist page to reach any of the collapsible panels. 
In Firefox, go to `Accessibility` tab in the dev-tools and check `Show Tabbing Order` to see the tab order of the elements on the page. Observe that both accordion panels are not in the tab order.
- Playlist items container does not have visible focus when it is reached using`Tab`. To observe this, focus on `Share` button and then press `Tab` key. Even though the focus is on the playlist items container, it is not visible. This is not reproducible in Ramp demo site.

### Done Looks Like
- [ ] A keyboard user can reach and expand/collapse the `Markers` panel
- [ ] A keyboard user can reach and expand/collapse the `Source Item Details` panel
- [ ] Focus is visible when use tab navigation to reach playlist items

**Comment by joncameron** [comment]
Works as expected except for the `Marker` and `Source item details` not getting the visible highlighting indicator. They get focus and are operational but there's no outline like when other elements on the page receive focus.

@Dananji is that an easy fix, or should I create a new issue for just the highlighting?
